### ISSUE TITLE
fluff in right leg critical slots

### ISSUE THREAD

**Issue by aflazmn** [juk0de/mtf2json#12]
keys from fluff like deployment, history, and capabilities are showing up as right leg slots 13 or 14. attached list of mechs that have this issue from megamek.
[right leg.txt](https://github.com/user-attachments/files/17042826/right.leg.txt)

**Comment by juk0de** [comment]
Thanks for reporting @aflazmn. The issue should be fixed in version `0.1.8`.

**Comment by aflazmn** [comment]
I discovered these additional mechs still have fluff showing up in right leg slots. There might still be more, but these ones have a period somewhere in the right leg.

Prey Seeker	PY-SR10
Hussar	HSR-200-D
Hussar	HSR-200-Db
Hussar	HSR-300-D
Hussar	HSR-350-D
Hussar	HSR-400-D
Hussar	HSR-500-D
Hussar	HSR-900-D
Hussar	HSR-950-D
Gyrfalcon	
Gyrfalcon	2
Gyrfalcon	3
Gyrfalcon	4
Blood Reaper	
Dragon Fire	DGR-3F
Hatamoto-Chi	HTM-26T
Hatamoto-Chi	HTM-27T
Hatamoto-Chi	HTM-27T (Daniel II)
Hatamoto-Chi	HTM-27T (Daniel)
Hatamoto-Chi	HTM-28T
Hatamoto-Chi	HTM-28T (Shin)
Hatamoto-Chi	HTM-28Tr
Hatamoto-Hi	HTM-27U
Hatamoto-Hi	HTM-C
Hatamoto-Hi	HTM-CM
Hatamoto-Kaeru	HTM-35K
Hatamoto-Kaeru	HTM-35X
Hatamoto-Kaze	HTM-27V
Hatamoto-Ku	HTM-27W
Hatamoto-Mizo	HTM-27Y
Spartan	SPT-N3
Spartan	SPT-NF
Alpha Wolf	A
Alpha Wolf	B
Alpha Wolf	C
Alpha Wolf	Prime
Mauler	MAL-1K
Mauler	MAL-1KX
Mauler	MAL-1PT5
Mauler	MAL-1PT6
Mauler	MAL-1R
Mauler	MAL-1X-AFFC
Mauler	MAL-1Y
Mauler	MAL-2D
Mauler	MAL-2R
Mauler	MAL-3K
Mauler	MAL-3R
Mauler	MAL-4R
Mauler	MAL-4X 'Todesbote'
Mauler	MAL-C
Amarok	
Amarok	2
Amarok	3
Thunder Hawk	TDK-7S

**Comment by juk0de** [comment]
That looks like a different issue. They've added lines to the affected MTF files that contain no key. Here's the "Amarok 3":
```
capabilities:<p>Boasting a mass more than doubled, a sizable portion is dedicated to its formidable arsenal. The ER PPC, amplified by a capacitor, serves as a long-range deterrent against incoming Mechs, while a rotary autocannon and large pulse laser offer reliable secondary firepower, complemented by twin Streak missile launchers.</p>
<p>An innovative feature, inspired by Solaris VII data, employs a pre-programmed routine to lock the arms above the cockpit, potentially thwarting DFA assaults. However, certain technical glitches have led to disabling this feature by many Amarok pilots, with plans to omit it in future productions. The Amarok's demand remains unaffected, as warriors vie for possession through Trials.</p>
``` 
Note that the line starting with `<p>An innovative feature` has no key at all!

That may actually be a bug in the MTF files. Although that's hard to say, since MTF is not a real file format and has no specification... I'll create a new ticket.

**Comment by juk0de** [comment]
> I discovered these additional mechs still have fluff showing up in right leg slots. There might still be more, but these ones have a period somewhere in the right leg.

@aflazmn: that issue should be fixed in version `0.2.0`
